### DISCUSSION TITLE
Ödev Hakkında

### DISCUSSION THREAD

**Discussion by Frost-White** [gusanmaz/BMB311_OS_2023#2] [Q&A] [answered]
Hocam hangi konuların dahil edilmeyeceğini yazabilirseniz çok sevinirim.

**Comment by gusanmaz** [comment, marked as answer]
6. Vi Text Editor (https://ryanstutorials.net/linuxtutorial/vi.php) konusu opsiyonel diger tum konular 1. proje kapsami icinde.
